mitxpro/templates/footer.html

```diff
@@ -11,21 +35,21 @@
             </div>
             <div class="right-info">
                 <ul class="footer-nav">
-                    <li><a href="#">About Us</a></li>
-                    <li><a href="#">FAQs</a></li>
-                    <li><a href="#">Blog</a></li>
-                    <li><a href="#">Contact Us</a></li>
+                    <li><a href="/about-us/" target="_blank">About Us</a></li>
+                    <li><a href="/blog/" target="_blank">Blog</a></li>
+                    <li>
+                        <form action="#" class="subscription-form">
+                            <input type="email" placeholder="Sign Up for Email Updates">
+                            <button type="submit" class="icon-arrow-circle-right"></button>
+                        </form>
+                    </li>
```

Comment:
If not this time, may be in future, if we can use https://docs.wagtail.io/en/v2.4/topics/snippets.html, then we can solve problem of hardcoding of these URLs.
@pdpinch Your thoughts ?